tools/api-designer/README.md

```diff
@@ -0,0 +1,36 @@
+# RAPID ROCKS API DESIGNER
+
+An experimental designer for RAPID/ODATA APIs and RSDL-based services.
+
+![Screenshot](./screenshot.jpeg)
+
+## Build instructions
+
+### Build related tools
+
+This project depends on other tools in this repo. You have to manually build those tools first by going to their respective directories and following their build instructions (mainly a combination of `npm install` + `npm run build`):
+- [odataUri](../odataUri) (referenced as `odata-uri` in `package.json`)
```

Comment:
Actually it is _not_ referenced in `package.json` - should it be?

Reply:
This doesn't have to be referenced in `api-designer` since it's referenced indirectly by `odata-url-editor`. I will change the wording t reflect that